var/spack/repos/builtin/packages/py-zmq/package.py

```diff
@@ -11,6 +11,7 @@ class PyZmq(PythonPackage):
     homepage = "https://github.com/zeromq/pyzmq"
     url      = "https://github.com/zeromq/pyzmq/archive/v14.7.0.tar.gz"
 
+    version('18.1.0', sha256='32f7618b8104021bc96cbd60be4330bdf37b929e8061dbce362c9f3478a08e21')
     version('17.1.2', sha256='77a32350440e321466b1748e6063b34a8a73768b62cb674e7d799fbc654b7c45')
     version('16.0.2', sha256='717dd902c3cf432b1c68e7b299ad028b0de0d0a823858e440b81d5f1baa2b1c1')
     version('14.7.0', sha256='809a5fcc720d286c840f7f64696e60322b5b2544795a73db626f09b344d16a15')
```

Comment:
For the `zeromq` dependency (which will be renamed `libzmq` in #13411), it looks like the minimum required version if 3.2. Older versions can be used, but it requires you to explicitly enable legacy support. Can you add the 3.2+ version constraint?

Reply:
Also, the minimum cython version is now 0.20

Reply:
And Python is 2.7+, 3.3+

Reply:
`py-py` and `py-cffi` may only be needed when building with PyPy? Can you test this theory?

Reply:
mh, `py-cffi` is referenced in the source code in `backend/cffi` but it build fine for me, although I haven't tested it in practice. I added a comment but would opt to keep the dependencies as I haven't actually tested it